rapidast.py

```diff
@@ -202,6 +209,81 @@ def dump_rapidast_redacted_configs(main_config_file_location: str, destination_d
             sys.exit(2)
 
 
+def deep_traverse_and_replace(d: dict, suffix: str) -> dict:  # pylint: disable=C0103
```

Comment:
very minor neatpick: since `suffix` will, in the current state, never be called with anything other than `_from_var`, I have the feeling that `deep_traverse_and_replace` should only take the dictionary as parameter, and have the `suffix` hardcoded in the function, and possibly renamed something like `deep_traverse_and_replace_with_var_content`.

-or- alternatively, if we might want to extend `_from_var` to other sources (e.g.: `_from_file`) : both suffix + replacement function could be added as parameter [ie.: `def deep_traverse_and_replace(d: dict, suffix=`_from_var`, function=lambda x: os.environ[x])` ]. I am not sure whether that's easily feasible

Reply:
Good point, I've made the change. I believe the first option will make the code clearer and more specific to rapidast case